Program.cs

```diff
@@ -1,10 +1,13 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// To print the factorial of a given number.
+
+Console.WriteLine (factorial ());
+string factorial () {

```

Comment:
Capital letter for function name. Factorial is a math the return type to be int. Do the string manipulation based on the result.